src/lib/draw/GeometryMode.svelte

```diff
@@ -124,6 +124,21 @@
     });
   }
 
+  // Auto-save
+  for (let tool of [polygonTool]) {
+    tool.addEventListenerUpdated((feature) => {
+      if ($currentMode == thisMode) {
+        gjScheme.update((gj) => {
+          let updateFeature = gj.features.find(
```

Comment:
I think a name like `featureToBeUpdated` would be slightly cleary

Reply:
This is from an intermediate commit; this file has been changed since then. I've made the equivalent change in the latest version though